pkg/runtime/podman_test.go

```diff
@@ -0,0 +1,56 @@
+package runtime
+
+import (
+	"context"
+	"flag"
+	"github.com/containers/libpod/v2/pkg/bindings/images"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+var testPodman = flag.Bool("podman", false, "run podman tests")
+
+func image_exists(conn context.Context, imageName string) bool {
+	exists, err := images.Exists(conn, imageName)
+	if err != nil {
+		panic(err)
+	}
+	return exists
+}
+
+// This test takes a long time to execute. Around 14 second on my computer.
+func TestPullImageNoCredentials(t *testing.T) {
+	if !*testPodman {
+		return
+	}
+
+	// XXX find a smaller image
```

Comment:
alpine is around 6 megs:
```
✗ docker images | grep alpine
alpine                                                                     latest                                     d6e46aa2470d   2 months ago    5.57MB
alpine                                                                     3.9                                        78a2ce922f86   8 months ago    5.55MB
```

Reply:
docker.io/library/hello-world is only 6kb ;-)